src/schema/nmdc.yaml

```diff
@@ -299,16 +299,23 @@ classes:
 
   FunctionalAnnotationAggMember:
     class_uri: nmdc:FunctionalAnnotationAggMember
+    unique_keys: 
+      main:
+        description: A FunctionalAnntoationAggMember record is uniquely identified by the combination of the was_generated_by and gene_function_id slots.
+        unique_key_slots:
+          - was_generated_by
+          - gene_function_id
     slots:
-      - metagenome_annotation_id
+      - was_generated_by
       - gene_function_id
       - count
       - type
     slot_usage:
-      metagenome_annotation_id:
-        structured_pattern: # doesn't include act
-          syntax: "{id_nmdc_prefix}:(wfmgan|wfmtan)-{id_shoulder}-{id_blade}{id_version}$"
+      was_generated_by:
```

Comment:
@aclum and others: are we simplifying the range of [was_generated_by](https://microbiomedata.github.io/nmdc-schema/was_generated_by/) at some point?

Reply:
See https://github.com/microbiomedata/nmdc-schema/pull/2246